charts/kyverno-policies/templates/restricted/disallow-capabilities-strict.yaml

```diff
@@ -111,7 +111,7 @@ spec:
             deny:
               conditions:
                 all:
-                - key: "{{`{{ element.securityContext.capabilities.add[] || '' }}`}}"
+                - key: "{{`{{ element.securityContext.capabilities.add[].to_upper(@) || `[]` }}`}}"
```

Comment:
new \` signs are confusing helm :(

Reply:
How do we fix? I'm not a Helm wizard. The two brackets need to be enclosed in backticks to be interpreted correctly.

Reply:
@chipzoller I pushed the fix in your fork

Reply:
Thanks.